src/components/MainMenu/Menu.jsx

```diff
@@ -2,15 +2,31 @@ import classes from "./Menu.module.scss";
 // import AddNewElemModal from "../AddNewElemModal/AddNewElemModal";
 import AddNewElem from "../AddNewElemModal/AddNewElem";
 
-const Menu = ({ onAddItem, items }) => {
+const Menu = ({ onAddItem, items, onItemClick, activeListIndex }) => {
   //GUARDA BENE LA STRUTTURA QUI SOTTO!!!!!!!!!!
+
   const listedItems = (
     <ul className={classes.list}>
-      {items.map((item) => (
-        <li key={item.id} className={classes.elem}>
-          {item.name}
-        </li>
-      ))}
+      {items.map(
+        (
+          item,
+          key // la key/index é un valore che quando mappi viene assegnato automaticamente a
+          //ogni singolo elem che viene iterato (come la "i" nel ciclo for)
+        ) => (
+          <li
+            key={`list-${key}`}
+            className={
+              activeListIndex === key ? classes.selected : classes.elem

```

Comment:
ecco l'errore: la classe `elem` dovrebbe essere sempre presente. La classe da applicare in maniera condizionale è solo `selected`